### DISCUSSION TITLE
Hide Ignored users on right side of pane

### DISCUSSION THREAD

**Discussion by Konpaku-LVX** [element-hq/element-meta#797] [Ideas]
#### What would you like to do?
when you ignore a user, it hides all their messages and stuff, but you can still see their name on the side, some users may want this or be ok with it, some might not, personally i would have specific use cases

#### Why would you like to do it?
for example if someone has an inappropriate name in a chat and you dont want to look at it on the right side panel, you could hide them from showing up on the users list
#### How would you like to achieve it?
when you press ignore, also hide them from the list, or if that may inconvenience some users, maybe make it a toggle in the settings to also hide ignored users on the side

### Have you considered any alternatives?

i tried using inspect element and other addons, but they dont work that well and you cant really in the desktop client

### Additional context

i just think that when someone ignores/blocks/whichever term you want to call it, most people would probably prefer not to see anything about them at all, or even be able to know they exist at all while they are ignored.

**Comment by t3chguy** [comment]
Ignore is actually implemented by the server

**Comment by Konpaku-LVX** [comment]
ok, so lets say i ignore user "Mario" in a group chat of 6 people, but i dont want to even see "Mario" on the right side of the pane with all the other users.


are you saying this is already implemented? because this is the feature i want, and i do not see it. if it is in element can you tell me how to use it?

 [Ignore is actually implemented by the server]
i know ignore is already in element. thats not what im requesting here. please read carefully

**Comment by t3chguy** [comment]
Element is a client, not a Server. The software actually filtering out ignored users' messages etc is your Matrix server, e.g. Synapse or Dendrite

**Comment by Konpaku-LVX** [comment]
i know that, all i want is a feature to hide ignored users so you dont have to see them, is that not understandable or something?

**Comment by t3chguy** [comment]
It is, hence this request remains open. It was moved here as it isn't for Element clients to implement, but for servers to implement

**Comment by Konpaku-LVX** [comment]
ok, thank you for understanding. it is appreciated. the main reason really if you are curios, is because people genuinely have names sometimes i just don't want to look at. 

heres one example
![image](https://user-images.githubusercontent.com/16366107/199486602-0aa593bd-a7b3-4786-951f-6717d5fd8923.png)

some alternatives i discovered if anyone is facing a similar issue is, 
1 set the side panel to like something other than the users
2. if you use the browser you can use extensions to change their name as soon or even before the page even loads fully. 
3. if you know how to you could code your own patch or something, 

usually i use 1 for client and 2 for browser
